### DISCUSSION TITLE
Just let me use the terminal colours.

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#6110] [Q&A] [answered]
I like my editor theme to match my terminal theme and I'm using a terminal that doesn't have theme options. I don't need fancy million-colour highlighting; just let helix use the colours that the terminal provides, not that eye-searing purple. How do I do that?

**Comment by kirawi** [comment, marked as answer]
You can use `:theme base16-terminal`, or use a modified version of it to get it closer to what you want.

**Reply by ghost** [reply]
This works, thanks. I wouldn't have thought to use base16 when I have no base16 on my system, but I guess it makes sense.

**Reply by askreet** [reply]
The 16 in `base16` refers to the ANSI standard 16 color palette (8 colors + bold) used by your terminal emulator by default. 256 and true color modes are extensions to that standard.

**Reply by mpstaton** [reply]
THANK YOU!

**Comment by gabydd** [comment]
The ``base16_terminal`` and ``base16_default`` use your terminal colours

**Reply by dessalines** [reply]
This is close, however the gutter ignores your terminal colors, and just uses a black background even on light themes.

**Comment by smason** [comment]
I just stumbled on the `term16_dark` and `term16_light` themes.  These seem to work better with my terminal's colors than the `base16` ones.

See the original [pull request](https://github.com/helix-editor/helix/pull/9477) for screenshots.

**Reply by dessalines** [reply]
Thx for this, it works great!
